src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/CaptureFarmingGear.kt

```diff
@@ -90,6 +93,16 @@ class CaptureFarmingGear {
             storage.pumpkinFortune = !storage.pumpkinFortune
             LorenzUtils.chat("Toggled expired pumpkin fortune to: ${storage.pumpkinFortune}")
         }
+
+        private fun getUniqueVisitorsForTier(tier: Int, progress: Int): Int {
```

Comment:
I think its more clear if we remove progress from this function and add it at the function call instead. The function name is misleading otherwise